devenv/dev-dashboards/panel-table/table_tests_new.json

```diff
@@ -451,7 +442,7 @@
         "y": 18
       },
       "id": 3,
-      "maxDataPoints": "10",
```

Comment:
Manually changed this. i don't know how it's possible that we produce a string here, but either the schema should be updated (eww, ick, please no) or TS updated to ensure this is never allowed to be a string. tbqh, i'm not sure how this got allowed in the first place, given that we're using the generated type here...